commit_parser.go

```diff
@@ -344,6 +355,28 @@ func (p *commitParser) uniqMentions(mentions []string) []string {
 	return arr
 }
 
+func (p *commitParser) processJiraIssue(commit *Commit, issueId string) {
+	issue, err := p.jiraClient.GetJiraIssue(commit.JiraIssueId)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to parse Jira story %s: %s\n", issueId, err)
```

Comment:
Should we be using the Logger here instead of `fmt` and `os`? https://github.com/git-chglog/git-chglog/blob/master/cmd/git-chglog/logger.go#L34